tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```
@@ -150,5 +150,5 @@ spec:
         - "$(params.STRICT)"
         - "-e"
         - >
-          .result == "SUCCESS" or ($strict | not)
+          .result != "FAILURE" or ($strict | not)
```

It's also possible to have an `ERROR` state, which we never set but mentioning it here just in case.

---

We could check for all the known passing statuses:
```suggestion
          .result == "SUCCESS" or or .result == "WARNING" or .result == "SKIPPED" or ($strict | not)
```